### PR TITLE
Update winrm-cannot-process-request.md

### PR DESCRIPTION
**Pull request by joinimran**
Added an extra cause and its solution.
Problem : https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/277

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9  - kindly do the copy editor review for this PR. Thanks.

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ PSRemotingTransportException
 + FullyQualifiedErrorId : PSSessionOpenedFailed
 ```
 
-## Cause
+## Cause NO 1
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:36]:
```suggestion
## Cause No. 1
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 2]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
     > [!NOTE]
     > If the service was already started but it's not responding, you may have to click **Restart**.
 6. Try to connect to Exchange Online again.
+## Cause NO 2
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:56]:
```suggestion
## Cause No. 2
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 3]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
 
+Proxy has been configured but proxy is not running
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:58]:
```suggestion
The proxy has been configured but the proxy is not running.
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 4]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+
+## Solution
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:60]:
```suggestion
### Solution
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 5]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+1.Take the steps to start the proxy and try again. OR
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:61]:
```suggestion
1.Take the steps to start the proxy and try again, OR
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 6]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+2.Determine current proxy by executing((in powershell run as administrator)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:62]:
```suggestion
2. Determine the current proxy by executing ((in PowerShell run as administrator):
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 7]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+  ```powershell
+     netsh winhttp show proxy
+   ```
+3.From here,you can reset the proxy settings
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:66]:
```suggestion
3. From here, you can reset the proxy settings:
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 8]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+  ```powershell
+     netsh winhttp set proxy "127.0.0.1:8888)
+  ```
+5.OR you can import the proxy settings from the internet explorer( assuming you are able to access the internet from the Internet Explorer)
+  ```powershell
+     netsh winhttp import proxy source=ie
+  ```
+
+  
+
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:80]:
```suggestion
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 9]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+   ```
+3.From here,you can reset the proxy settings
+  ```powershell
+     netsh winhttp reset proxy
+  ```
+4.OR you can change to proxt something else(in the example 127.0.0.1:8888)
+  ```powershell
+     netsh winhttp set proxy "127.0.0.1:8888)
+  ```
+5.OR you can import the proxy settings from the internet explorer( assuming you are able to access the internet from the Internet Explorer)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:74]:
```suggestion
5. OR you can import the proxy settings from Internet Explorer (assuming you are able to access the Internet from Internet Explorer):
```

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 10]
```diff
@@ -53,9 +53,33 @@ To resolve this issue, check whether the Windows Remote Management service is in
...
+1.Take the steps to start the proxy and try again. OR
+2.Determine current proxy by executing((in powershell run as administrator)
+  ```powershell
+     netsh winhttp show proxy
+   ```
+3.From here,you can reset the proxy settings
+  ```powershell
+     netsh winhttp reset proxy
+  ```
+4.OR you can change to proxt something else(in the example 127.0.0.1:8888)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:70]:
```suggestion
4. OR you can change proxt to something else (in this example 127.0.0.1:8888):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md [hunk 11]
```diff
@@ -53,9 +53,31 @@ To resolve this issue, check whether the Windows Remote Management service is in
     > [!NOTE]
     > If the service was already started but it's not responding, you may have to click **Restart**.
 6. Try to connect to Exchange Online again.
+## Cause No. 2
+
+The proxy has been configured but the proxy is not running.
+
+### Solution
+1.Take the steps to start the proxy and try again, OR
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/winrm-cannot-process-request.md:61]:
```suggestion
1. Take the necessary steps to start the proxy and try again, OR

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @joinimran, one more.

**Comment by joinimran** [comment]
Thanks, @JohanFreelancer9 for all your input. You are the master.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
Thanks @joinimran!

**Comment by helenclu** [comment]
Can't edit the file, updated through private PR 2861, close this one.

**Pull request closed without merging**